tclock.go

```diff
@@ -443,7 +443,7 @@ func Main() int { //nolint:funlen,gocognit,gocyclo,maintidx // we could split th
 			if n > 0 {
 				ap.MoveCursor(prevX, prevY) // back to where we left off
 				_, _ = writer.Write(buf[:n])
-				prevX, prevY, _ = ap.ReadCursorPosXY()
+				// prevX, prevY, _ = ap.ReadCursorPosXY()
```

Comment:
that wouldn't work very well... as in the file wouldn't be tailed for new input

the fix was https://github.com/fortio/terminal/pull/170

but also changing how this is done in #45